README.md

```diff
@@ -133,15 +133,6 @@ In the **Bastion** account, create a group called `assume-read` with the policy:
       "Effect": "Allow",
       "Action": [ "sts:AssumeRole" ],
       "Resource": [ "arn:aws:iam::123456789012:role/read" ],
-      "Condition": {
```

Comment:
I dont think you should change the README example given it is "best-practice". Deleting example code is must easier than recreating it.